lib/include/kframework/avm/teal/teal-stack.md

```diff
@@ -55,6 +55,25 @@ The elements of the stack are values of sort `TValue`, i.e. either `TUInt64` or
                                                    andBool N  <Int #sizeTStack(XS)
 ```
 
+- `S { N }` accesses element $N$ (counting from the bottom of the stack) of $S$.
+```k
+  syntax TValue ::= TStack "{" Int "}" [function]
+  // -----------------------------------------------------------
+  rule (X : XS):TStack { N } => X
+    requires N ==Int #sizeTStack(XS)
+  rule XS              { N } => #drop((#sizeTStack(XS) -Int N) -Int 1, XS) [ 0 ]
+    requires N  <Int #sizeTStack(XS)
+
+//  rule XS:TStack {N} => #sizeTStack(#drop(#sizeTStack(XS) -Int N, XS))
```

Comment:
Why is this line commented out? Should we just drop it?

Reply:
I removed it.